test/rfxcom.spec.js

```diff
@@ -466,14 +606,14 @@ describe("RfxCom", function() {
             describe("device type identification", function() {
                 it("should identify HomeEasy EU devices", function(done) {
                     device.on("lighting2", function(evt) {
-                        expect(evt.subtype).toBe(rfxcom.lighting2.HOMEEASY_EU);
+                        expect(evt.subtype).toBe(rfxcom.lighting2[1]);
```

Comment:
The trouble is, the current code is not consistent. In a lighting1 event evt.subtype is a string, whereas in a lighting2 event, evt.subtype is a number. How about leaving evt.subtype exactly as it is for backward compatibility, but adding two new properties evt.subtypeNumber and evt.subtypeName?